### PR TITLE
Drop working directory when we're already there

### PR DESCRIPTION
**Pull request by timja**
mayb? @olblak

### CODE REVIEW

**Comment by olblak** [comment]
The reason why I wanted to use working_dir was because I though it contained the full path which would be more robust than using relative path.

**Comment by timja** [comment]
> The reason why I wanted to use working_dir was because I though it contained the full path which would be more robust than using relative path.

WORKING_DIR is only the relative path currently, if you want to do that then that part needs fixing up, root of the repo should be safe though?

**Comment by timja** [comment]
https://github.com/jenkins-infra/release/blob/b45a9d96f850bd54156825503a01d6f8ccb6b665/Jenkinsfile.d/package#L50

**Comment by olblak** [comment]
> > The reason why I wanted to use working_dir was because I though it contained the full path which would be more robust than using relative path.
> 
> WORKING_DIR is only the relative path currently, if you want to do that then that part needs fixing up, root of the repo should be safe though?

or we could use the variable WORKSPACE defined by Jenkins

File: utils/release.sh [hunk 1]
```diff
@@ -261,12 +261,12 @@ function configurePackagingEnv(){
 
   requireGPGPassphrase
 
-  : "${BRAND:=$WORKING_DIRECTORY/branding/common}"
+  : "${BRAND:=branding/common}"
```
Comment (olblak) [inline utils/release.sh]:
```suggestion
  : "${BRAND:=$WORKSPACE/$WORKING_DIRECTORY/branding/common}"
```

File: utils/release.sh [hunk 2]
```diff
@@ -261,12 +261,12 @@ function configurePackagingEnv(){
...
   : "${RELEASELINE:=}"
   : "${ORGANIZATION:=jenkins.io}"
-  : "${BUILDENV:=$WORKING_DIRECTORY/env/release.mk}"
+  : "${BUILDENV:=env/release.mk}"
```
Comment (olblak) [inline utils/release.sh]:
```suggestion
  : "${BUILDENV:=$WORKSPACE/$WORKING_DIRECTORY/env/release.mk}"
```

File: utils/release.sh [hunk 3]
```diff
@@ -261,12 +261,12 @@ function configurePackagingEnv(){
...
   : "${CREDENTIAL:=$BRAND}" # For now, we just want this variable to be set to not empty
-  : "${GPG_PASSPHRASE_FILE:=$WORKING_DIRECTORY/$GPG_KEYNAME.pass}"
+  : "${GPG_PASSPHRASE_FILE:=$GPG_KEYNAME.pass}"
```
Comment (olblak) [inline utils/release.sh]:
```suggestion
  : "${GPG_PASSPHRASE_FILE:=$WORKSPACE/$WORKING_DIRECTORY/$GPG_KEYNAME.pass}"
```

**Pull request merged**